src/hooks/screen/useChatScreenHook.ts

```diff
@@ -313,6 +318,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
   React.useEffect(() => {
     return () => {
       setChats([]);
+      setSelectedChannel(null);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line is part of the cleanup function in a `useEffect` hook. The call to `setSelectedChannel(null)` is a good practice as it resets the state when the component unmounts, preventing potential memory leaks or stale data issues.

```typescript
setSelectedChannel(null);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->